### PR TITLE
docs(README): remove the build from source instructions

### PR DESCRIPTION
**Pull request by philips**
docs/developers.md already has these instructions. And 80% of the time
someone looking at this project is interested in getting started not
hacking directly.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/helm/1176)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Think of it like apt/yum/homebrew for Kubernetes.
 - Charts can be stored on disk, or fetched from remote chart repositories
   (like Debian or RedHat packages)
 
+## Install
+
+Download a [release tarball of helm and tiller for your platform](https://github.com/kubernetes/helm/releases). Unpack the `helm` and `tiller` binaries and add them to your PATH and you are good to go! OS X/[Cask](https://caskroom.github.io/) users can `brew cask install helm`.
```
Comment (adamreese) [inline README.md]: The tarball only includes helm binary now

Comment (philips) [inline README.md]: @adamreese will update.

**Comment by philips** [comment]
@adamreese all fixed.

**Comment by technosophos** [comment]
Seems to have #1175 rolled into the same PR.

**Comment by philips** [comment]
@technosophos yes, can we merge #1175 first? Trying to make atomic commits to docs is really hard. Really when re-orging things.

**Comment by philips** [comment]
rebased on master after #1175 merged

**Pull request merged**